app_inference_spec.py

```diff
@@ -0,0 +1,92 @@
+from fastapi import HTTPException
+from pydantic import BaseModel
+from models_host.base_inference_spec import BaseInferenceSpec
+from typing import List
+import detoxify
+import torch
+import os
+
+class InferenceData(BaseModel):
+    name: str
+    shape: List[int]
+    data: List
+    datatype: str
+
+
+class InputRequest(BaseModel):
+    inputs: List[InferenceData]
+
+
+class OutputResponse(BaseModel):
+    modelname: str
+    modelversion: str
+    outputs: List[InferenceData]
+
+# Using same nomencalture as in Sagemaker classes
+class InferenceSpec(BaseInferenceSpec):
+    model = None
+
+    model_name = "unbiased-small"
+    validation_method = "sentence"
+    labels = [
+        "toxicity",
+        "severe_toxicity",
+        "obscene",
+        "threat",
+        "insult",
+        "identity_attack",
+        "sexual_explicit",
+    ]
+
+    @property
+    def torch_device(self):
+        env = os.environ.get("env", "dev")
+        torch_device = "cuda" if env == "prod" else "cpu"
+        return torch_device
```

Comment:
If we're bound to this for historical reasons I don't oppose it, but I feel like names should be indicative of what they are.  Could we call the environment variable 'inference_device' and pass it directly?  If I read 'env' for something else dev but want to run on my local GPU I might make a bad mistake.

Reply:
this was left for reverse compatibility but now that we're fully switched over this is a good time to phase it out 👍🏼